src/LiveComponent/README.md

```diff
@@ -524,6 +538,26 @@ class RandomNumberComponent
 }
 ```
 
+### Customize Default Action
+
+The default component action is `__invoke()` by default. You
+can customize the method with the `AsLiveComponent` attribute:
+
+```php
+// src/Components/ProductSearchComponent.php
+namespace App\Components;
+
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+
+#[AsLiveComponent('product_search', defaultAction: 'defaultAction()')]
+class ProductSearchComponent
+{
+    public function defaultAction(): void
+    {
+    }
+}
+```
+
```

Comment:
Boy, I just can't think of a use-case for this. I'm inclined to not document it at all!

Reply:
I know, me either. I almost documented injecting services but again, couldn't think of a use case.

I'm in favor of axing this section. What do you think?

Reply:
💯 let's kill it, but obviously, keep the option.

Could you also add one bullet to the CHANGELOG for LiveComponent?